protos/processor.proto

```diff
@@ -24,6 +24,14 @@ import "transaction.proto";
 // The registration request from the transaction processor to the
 // validator/executor
 message TpRegisterRequest {
```

Comment:
Add a comment for this request describing the features that are included in protocol version 1.

Reply:
Done